usaspending_api/database_scripts/matviews/vacuum_matviews.sql

```diff
@@ -12,4 +12,5 @@ VACUUM ANALYZE VERBOSE summary_view_naics_codes;
 VACUUM ANALYZE VERBOSE summary_view_psc_codes;
 VACUUM ANALYZE VERBOSE universal_award_matview;
 VACUUM ANALYZE VERBOSE universal_transaction_matview;
-VACUUM ANALYZE VERBOSE tas_award_matview;
\ No newline at end of file
+VACUUM ANALYZE VERBOSE tas_autocomplete_matview;
+VACUUM ANALYZE VERBOSE tas_search_matview;
```

Comment:
Is it documented that you have to add every new matview to all these lists? Cause I had no idea.

Reply:
Not to my knowledge.  As part of due diligence, I just searched around for other materialized views and mimicked what I found where it made sense.

Reply:
Can you think of an appropriate place to do this, or if it's a good idea? I'm happy to do the work to capture this if it's not going to be ignored/useless immediately.

Reply:
Not everything can be documented (I mean Wikipedia tried, but I can't find anything for this piece of code there, so clearly they failed).  At some point, you have to rely on your skills as a software developer to actually look for and update code where necessary.

But I'm also not going to stand in your way if you want to write documentation.